src/test/javascript/spec/integration/code-editor/code-editor-container.integration.spec.ts

```diff
@@ -226,66 +212,66 @@ describe('CodeEditorContainerIntegration', () => {
         containerFixture.detectChanges();
 
         // container
-        expect(container.commitState).to.equal(CommitState.CLEAN);
-        expect(container.editorState).to.equal(EditorState.CLEAN);
-        expect(container.buildOutput.isBuilding).to.be.false;
-        expect(container.unsavedFiles).to.be.empty;
+        expect(container.commitState).toBe(CommitState.CLEAN);
+        expect(container.editorState).toBe(EditorState.CLEAN);
+        expect(container.buildOutput.isBuilding).toBe(false);
+        expect(container.unsavedFiles).toStrictEqual({});
 
         // file browser
-        expect(checkIfRepositoryIsCleanStub).to.have.been.calledOnce;
-        expect(getRepositoryContentStub).to.have.been.calledOnce;
-        expect(container.fileBrowser.errorFiles).to.deep.equal(extractedErrorFiles);
-        expect(container.fileBrowser.unsavedFiles).to.be.empty;
+        expect(checkIfRepositoryIsCleanStub).toHaveBeenCalledTimes(1);
+        expect(getRepositoryContentStub).toHaveBeenCalledTimes(1);
+        expect(container.fileBrowser.errorFiles).toEqual(extractedErrorFiles);
+        expect(container.fileBrowser.unsavedFiles).toBeArrayOfSize(0);
```

Comment:
```suggestion
        expect(container.fileBrowser.unsavedFiles).toHaveLength(0);
```
Does the regular Jest assertion produce different results here?
As the attribute is defined as `string[]`, I’d expect it to always be an array.

Reply:
Replaced 👍